examples/photoCalTask.py

```diff
@@ -75,19 +70,25 @@ def run():
     exposure, srcCat = loadData()
     schema = srcCat.getSchema()
     #
+    # Create the reference catalog loader
+    #
+    refCatDir = os.path.join(lsst.utils.getPackageDir('meas_astrom'), 'tests', 'data', 'sdssrefcat')
```

Comment:
The data look like they are in `pipe_tasks` not `meas_astrom`.  Does this example run with these changes?